numba_dppy/dpnp_glue/dpnp_array_ops_impl.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numba_dppy.dpnp_glue.dpnpimpl as dpnp_ext
+from numba import types
+from numba.core.typing import signature
+from . import stubs
+import numba_dppy.dpnp_glue as dpnp_lowering
+from numba.core.extending import overload
+import numpy as np
+from numba_dppy import dpctl_functions
+import numba_dppy
+
+
+@overload(stubs.dpnp.sort)
+def dpnp_sort_impl(a):
+    name = "sort"
+    dpnp_lowering.ensure_dpnp(name)
+
+    ret_type = types.void
+    """
+    dpnp source:
+    https://github.com/IntelPython/dpnp/blob/master/dpnp/backend/kernels/dpnp_krnl_sorting.cpp#L90
```

Comment:
Master branch will keep on changing, it is better to find the latest release and post the reference from there. Please look at other snippet, we have chosen the latest release of dpnp (which was 0.4.0) at that time.

Reply:
I agree that permalinks should be used. But let's use links from 0.5.0, since 0.4.0 has too much deprecated
And we almost switched to 0.5.0